.github/workflows/binary-compatibility-checks.yml

```diff
@@ -30,11 +30,11 @@ jobs:
           # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
           fetch-depth: 0
 
-      - name: Setup Java 11
+      - name: Setup Java 8
         uses: actions/setup-java@v3
         with:
           distribution: temurin
-          java-version: 11
+          java-version: 8
```

Comment:
our release process for this repo requires Java 11 for the build

Reply:
Ah right, I got this mixed up with other repos, let me revert